models/mednist_gan/configs/train.json

```diff
@@ -0,0 +1,168 @@
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "LoadImaged",
+                "keys": "image"
+            },
+            {
+                "_target_": "AddChanneld",
+                "keys": "image"
+            },
+            {
+                "_target_": "ScaleIntensityd",
+                "keys": "image"
+            },
+            {
+                "_target_": "RandRotated",
+                "keys": "image",
+                "range_x": "$np.pi/12",
+                "prob": 0.5,
+                "keep_size": true
+            },
+            {
+                "_target_": "RandFlipd",
+                "keys": "image",
+                "spatial_axis": 0,
+                "prob": 0.5
+            },
+            {
+                "_target_": "RandZoomd",
+                "keys": "image",
+                "min_zoom": 0.9,
+                "max_zoom": 1.1,
+                "prob": 0.5
+            },
+            {
+                "_target_": "EnsureTyped",
+                "keys": "image"
+            }
+        ]
+    },
+    "real_dataset": {
+        "_target_": "CacheDataset",
+        "data": "$[{'image': i} for i in @datalist]",
+        "transform": "@preprocessing"
+    },
+    "real_dataloader": {
+        "_target_": "DataLoader",
+        "dataset": "@real_dataset",
+        "batch_size": 600,
+        "shuffle": true,
+        "num_workers": 12
+    },
+    "doptimizer": {
+        "_target_": "torch.optim.Adam",
+        "params": "$@dnetwork.parameters()",
+        "lr": 0.0002,
+        "betas": [
+            0.5,
+            0.999
+        ]
+    },
+    "goptimizer": {
+        "_target_": "torch.optim.Adam",
+        "params": "$@gnetwork.parameters()",
+        "lr": 0.0002,
+        "betas": [
+            0.5,
+            0.999
+        ]
+    },
+    "handlers": [
+        {
+            "_target_": "CheckpointSaver",
+            "save_dir": "$@bundle_root + '/models'",
+            "save_dict": {
+                "model": "@gnetwork"
+            },
+            "save_interval": 0,
+            "save_final": true,
+            "epoch_level": true
+        }
+    ],
+    "trainer": {
+        "_target_": "GanTrainer",
+        "device": "@device",
+        "max_epochs": 50,
+        "train_data_loader": "@real_dataloader",
+        "g_network": "@gnetwork",
+        "g_optimizer": "@goptimizer",
+        "g_loss_function": "$partial(generator_loss, disc_net=@dnetwork)",
+        "d_network": "@dnetwork",
+        "d_optimizer": "@doptimizer",
+        "d_loss_function": "$partial(discriminator_loss, disc_net=@dnetwork)",
+        "d_prepare_batch": "$prepare_batch",
+        "d_train_steps": 5,
+        "g_update_latents": true,
+        "latent_shape": "@latent_size",
+        "key_train_metric": "$None",
+        "train_handlers": "@handlers"
+    },
+    "training": [
+        "$@gnetwork.conv.add_module('activation', torch.nn.Sigmoid())",
```

Comment:
I think you added this activation layer here because the loss function doesn't have activation arg?
Maybe it's better to still use the `postprocessing` arg?

Thanks.

Reply:
I had this here because it's what's in the original [notebook in the tutorials repo](https://github.com/Project-MONAI/tutorials/blob/main/modules/mednist_GAN_workflow_dict.ipynb). I thought that sticking to it in this way would be better so those familiar with the notebook could follow here where the same components go, and also as an example of modifying the network. The inference notebook uses the postprocessing activation, I can change that here still if you want.